trident/absorption_spectrum/absorption_spectrum.py

```diff
@@ -468,8 +482,10 @@ def make_spectrum(self, input_object, output_file=None,
         input_ds.domain_left_edge = input_ds.domain_left_edge.to('code_length')
         input_ds.domain_right_edge = input_ds.domain_right_edge.to('code_length')
 
-        if self.bin_space == 'velocity':
-            self.zero_redshift = getattr(input_ds, 'current_redshift', 0)
+        self.hubble_constant = getattr(input_ds,'hubble_constant')
```

Comment:
I fear that this may break functionality for simulations which aren't cosmological.  Notice for the redshift, there is a final argument that sets it to 0 in the case that there is no current_redshift parameter.  This is to ensure that the code works on non-cosmological datasets by just setting the current redshift value to 0.  

Could you group these parameter instantiations along with your `co = Cosmology` and `hnow` code below, and maybe put it inside an if/then block for deposition_method='delta', or a try/except block and give a default value for the z=0 case, so non-cosmological datasets will still run?